docs/de/programming/python/tutorial-1.md

```diff
@@ -0,0 +1,130 @@
+---
+nav-title: The First App
+nav-pos: 1
+---
+# Programmieren in Python: die erste Application
+
+Dieses Tutorial soll den Einstieg erleichtern und die Grundschritte erklären!
+
+Eine Application besteht aus **3** Teilen:
+
+ * Das **Programm**. Dies ist normalerweise ein Python-Script. Es könnte aber auch jede beliebige andere Programmiersprache sein Seit [Python](https://www.python.org) der Standart für die TXT-Programmierung ist wird dieses Tutorial sich ausschließlich darauf beziehen!
+
+ * Ein **Manifest**. Das ist eine kleine Datei die Den Programmnamen, Autor, ... angibt
+
+ * Ein **Icon**. Dieses sollte ein PNG-Bild sein mit 64x64 Pixeln. Es wird sowohl auf dem Betriebssystem als auch auf der Weboberfläche verwendet!
```

Comment:
"Dieses sollte ein PNG-Bild sein mit 64x64 Pixeln." -> "Das Icon sollte ein PNG-Bild mit 64x64 Pixeln sein."

"Es wird sowohl auf dem Betriebssystem als auch auf der Weboberfläche verwendet!" -> "Es wird sowohl auf dem Hauptbildschirm als auch in der Weboberfläche angezeigt." (ohne Ausrufezeichen &ndash; so sensationell ist das jetzt auch wieder nicht, dass wir das Icon irgendwo anzeigen :smile:)

"Hauptbildschirm" könnte man eventuell noch nach http://cfw.ftcommunity.de/ftcommunity-TXT/de/getting-started/usage.html verlinken.